docs/src/examples/components/Embed/Types/index.tsx

```diff
@@ -10,6 +10,16 @@ const Types = () => (
       description="A basic embedded GIF."
       examplePath="components/Embed/Types/EmbedExample"
     />
+    <ComponentExample
+      title="IFrame"
+      description="An embed component can embed any iframe."
+      examplePath="components/Embed/Types/EmbedExampleIframe"
+    />
+    <ComponentExample
+      title="YouTube"
+      description="An embed component can embed a YouTube video by iframe."
```

Comment:
Let's move this under `Usage` section.
And keep `Iframe` under `Slots`

Reply:
Both IFrame example and video example belong under Slots section right?

Reply:
Yep, no `Types`. It doesn't have sense there